src/SamplesApp/SamplesApp.UITests/UnoSamples_Tests.Keyboard.cs

```diff
@@ -443,13 +443,16 @@ public void Keyboard_DismissTesting()
 			float yPosition = appCommandBarPosition.Height / 2;
 			_app.TapCoordinates(appCommandBarPosition.X + xPosition, appCommandBarPosition.Y + yPosition);
 			_app.Wait(3);
-			_app.Back();
+			appCommandBar.FastTap();
```

Comment:
Why a change is needed for this test?

Reply:
In the master branch will fail unexpected as well, isen't for  100% of the tries, but will because the test doing Assertion Condition after send _app.Back(). I just moved "Back()" after the Assertion Condition to wait the ImageAssert do his work.

I removed the FastTap that I introduced, is not needed anymore.


![image](https://user-images.githubusercontent.com/5515535/161385627-92b2e371-1133-432b-94b2-4f58c12b860c.png)

![image](https://user-images.githubusercontent.com/5515535/161385661-a6dc8549-2dc1-47bb-ade3-878fabc9b9cf.png)